terraform/locals.tf

```diff
@@ -0,0 +1,8 @@
+locals {
+  tags = {
+    business-unit = "Platforms"
+    is-production = true
+    owner         = "Hosting Leads <hosting-leads@digital.justice.gov.uk>"
```

Comment:
Should this be `operations-engineering@digital.justice.gov.uk`

Reply:
Good spot!